docs/reference/manifest/runtimes.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Runtimes in the manifest file
 description: ''
-ms.date: 01/24/2020
+ms.date: 02/13/2020
 localization_priority: Normal
 ---
 
 # Runtimes element
 
-This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Should follow the `<Host>` element in your manifest file.
+This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Child of the `<Host>` element in your manifest file.
```

Comment:
Suggestion: Highlight that this (and other manifest elements) is in preview by using an alert (important or note). [Example](https://github.com/OfficeDev/office-js-docs-pr/commit/26754a60a94344b6f7f5e9834ebad34323ebb789#diff-fcf8cca2ad6b7b1c1a8e7725ded7fbd7)